From: Roland Hieber <r.hieber@pengutronix.de>
Date: Sun, 28 Jan 2018 22:53:59 +0100
Subject: [PATCH] Release pengutronix/multi_v7/20180128-1

Signed-off-by: Roland Hieber <r.hieber@pengutronix.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f77a1fd722b7..49fa5c7da870 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2018
 PATCHLEVEL = 01
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20180128-1
 NAME = None
 
 # *DOCUMENTATION*
