From: Sascha Hauer <s.hauer@pengutronix.de>
Date: Wed, 8 Jan 2020 11:15:45 +0100
Subject: [PATCH] Release
 2019.12.0/customers/pengutronix/distrokit-v7a/20200108-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 600ca36a311c..96890b4c13f6 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2019
 PATCHLEVEL = 12
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20200108-1
 NAME = None
 
 # *DOCUMENTATION*
