From: Michael Grzeschik <m.grzeschik@pengutronix.de>
Date: Fri, 10 Jun 2016 14:10:58 +0200
Subject: [PATCH] Release 2016.05.0/customers/pengutronix/multi_v7/20160610-1

Signed-off-by: Michael Grzeschik <m.grzeschik@pengutronix.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 93f1d17d99d2..b844aa674da5 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2016
 PATCHLEVEL = 05
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20160610-1
 NAME = None
 
 # *DOCUMENTATION*
