From: Rouven Czerwinski <r.czerwinski@pengutronix.de>
Date: Fri, 18 Jan 2019 11:36:33 +0100
Subject: [PATCH] Release 2019.01.0/customers/pengutronix/multi_v7/20190118-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d6423b84fef0..6617637ca045 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2019
 PATCHLEVEL = 01
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20190118-1
 NAME = None
 
 # *DOCUMENTATION*
