From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Mon, 3 Jul 2017 12:12:17 +0200
Subject: [PATCH] Release 2017.07.0/pengutronix/multi_v7/20170703-1

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 35158319a601..10d2335d4389 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2017
 PATCHLEVEL = 07
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20170703-1
 NAME = None
 
 # *DOCUMENTATION*
