From: Roland Hieber <r.hieber@pengutronix.de>
Date: Tue, 15 May 2018 23:44:48 +0200
Subject: [PATCH] Release pengutronix/multi_v7/20180515-1

Signed-off-by: Roland Hieber <r.hieber@pengutronix.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 88c19917394f..11e79cc339f4 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2018
 PATCHLEVEL = 05
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20180515-1
 NAME = None
 
 # *DOCUMENTATION*
