From: Marco Felsch <m.felsch@pengutronix.de>
Date: Fri, 10 Feb 2023 12:51:49 +0100
Subject: [PATCH] Release 2.8/customers/pengutronix/distrokit/20230210-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 1bfff716485a..cf80e5cac8f3 100644
--- a/Makefile
+++ b/Makefile
@@ -327,7 +327,7 @@ endif
 ifeq (${BUILD_STRING},)
         BUILD_STRING  :=  $(shell git describe --always --dirty --tags 2> /dev/null)
 endif
-VERSION_STRING    :=  v${VERSION}(${BUILD_TYPE}):${BUILD_STRING}
+VERSION_STRING    :=  v${VERSION}(${BUILD_TYPE}):${BUILD_STRING}-20230210-1
 
 ifeq (${AARCH32_INSTRUCTION_SET},A32)
 TF_CFLAGS_aarch32	+=	-marm
