From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Tue, 22 Aug 2017 18:26:22 +0200
Subject: [PATCH] Release 2017.08.0/pengutronix/multi_v7/20170822-1

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ab44a1383365..eabe0ad5dc59 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2017
 PATCHLEVEL = 08
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20170822-1
 NAME = None
 
 # *DOCUMENTATION*
