From: Sascha Hauer <s.hauer@pengutronix.de>
Date: Fri, 21 Aug 2020 07:43:53 +0200
Subject: [PATCH] Release v2020.08.1-imx8m/20200821-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d27b78201bda..d6aab4c98507 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2020
 PATCHLEVEL = 08
 SUBLEVEL = 1
-EXTRAVERSION =
+EXTRAVERSION =-20200821-1
 NAME = None
 
 # *DOCUMENTATION*
