From: =?UTF-8?q?Uwe=20Kleine-K=C3=B6nig?= <u.kleine-koenig@pengutronix.de>
Date: Wed, 14 Jun 2017 22:22:27 +0200
Subject: [PATCH] Release 2017.06.0/pengutronix/multi_v7/20170614-1
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Uwe Kleine-König <u.kleine-koenig@pengutronix.de>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 42948fafa33f..284a039e61db 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2017
 PATCHLEVEL = 06
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20170614-1
 NAME = None
 
 # *DOCUMENTATION*
